test/intl402/NumberFormat/prototype/format/format-rounding-increment-1.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-intl.numberformat.prototype.format
+description: When set to `1`, roundingIncrement is correctly applied
+features: [Intl.NumberFormat-v3]
+includes: [testIntl.js]
+---*/
+
+var locales = [
+  new Intl.NumberFormat().resolvedOptions().locale, 'ar', 'de', 'th', 'ja'
+];
+var numberingSystems = ['arab', 'latn', 'thai', 'hanidec'];
+
+testNumberFormat(
+  locales,
+  numberingSystems,
+  {roundingIncrement: 1, maximumFractionDigits: 1},
+  {
+    '1.100': '1.1',
```

Comment:
Nit: Here and elsewhere: add a test for `'1.01': '1'` (soon to be `1.0`) since trailing zero behavior is important.